docs/pages/get-started/installation.mdx

```diff
@@ -4,42 +4,48 @@ title: Installation
 
 import { Terminal } from '~/ui/components/Snippet';
 
-To develop applications with Expo, you need two tools. A command-line tool called [Expo CLI](#1-expo-cli) to serve your project, and a mobile client app called [Expo Go](#2-expo-go-app-for-ios-and) to open the project on iOS and Android platforms. Additionally, you can use any web browser to run the project on the web.
+To develop applications with Expo, you need two tools. A command-line tool called [Expo CLI](#1-expo-cli) to serve your project,
+and a mobile client app called [Expo Go](#2-expo-go-app-for-ios-and) to open the project on iOS and Android platforms.
+Additionally, you can use any web browser to run the project on the web.
 
-> You don't need macOS to build an iOS app with Expo. You only need an iOS device to run the Expo Go app. Windows, Linux, and macOS are all supported for your development machine.
+> **info** You don't need macOS to build an iOS app with Expo. You only need an iOS device to run the Expo Go app. Windows, Linux, and macOS are all supported for your development machine.
 
 ## 1. Expo CLI
 
-[Expo CLI](/workflow/expo-cli) is a command-line tool that is the primary interface between a developer and other Expo tools. You are going to use it for different tasks in the development life cycle of your project such as serving the project in development, viewing logs, opening the app on an emulator or a physical device, and so on.
+[Expo CLI](/workflow/expo-cli) is a command-line tool that is the primary interface between a developer and other Expo tools.
+You are going to use it for different tasks in the development life cycle of your project such as serving the project in development, viewing logs, opening the app on an emulator or a physical device, and so on.
 
 ### Requirements
 
 To use Expo CLI, you need to have the following tools installed on your developer machine:
 
 - [Node.js LTS release](https://nodejs.org/en/)
+  > Only Node.js LTS releases (even-numbered) are recommended. As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS or Maintenance LTS releases.".
```

Comment:
Nesting the callout here doesn't look good IMO. The spacing looks odd and also there is no margin between the bullet point and the callout.

<img width="899" alt="CleanShot 2022-10-29 at 13 31 22@2x" src="https://user-images.githubusercontent.com/10234615/198820827-77853b94-6026-442f-a708-02605894bed4.png">

Also, there is no need for double periods at the end of the callout: `Maintenance LTS releases.".` can be `Maintenance LTS releases".`

Reply:
What do you think about ditching the callout and leaving the content as a part of the bullet (like description)?

Reply:
Yeah, i think making it part of the bullet sounds better.